HW_3/task_2.py

```diff
@@ -24,7 +24,6 @@
 print(a)
 Multy = []
 size = math.ceil(number / 2)
-
 def product_of_list(a):
     for i in range(size):
             Multy = [a[i] * a[-1-i] for i in range(0,i + 1)]
```

Comment:
Ирина тут либо одно, либо другое. Зачем вы внутри цикла еще раз создаете генератор списков? Причём этот генератор считается каждую итерацию цикла.
```suggestion
    Multy = [a[i] * a[-1-i] for i in range(size)]
```